README.md

```diff
@@ -16,6 +35,280 @@ ember install @bagaar/ember-permissions
 
 ## Usage
 
+### 1\. Setting and Checking Permissions for the Current Session
```

Comment:
I would shorten this. People usually just skim the readme until they find what they need.

Maybe something like `Setting up user session permissions` or `Setup user session permissions`